index.js

```diff
@@ -317,13 +339,24 @@ async function loadUserNotes(user, start, end) {
         users = user.split(",").filter(x => x);
     }
 
-    if (!end) {
-        end = Date.now() / 1000; // ms -> sec
+    if (end) {
+        end = parseInt(end, 10);
+    } else {
+        end = Math.floor(Date.now() / 1000); // ms -> sec
     }
-    if (!start) {
+    if (start) {
+        start = parseInt(start, 10);
+    } else {
         start = end - LOOKBACK_SECONDS;
     }
 
+    if (end < start) {
+        end = start;
+    }
+    if (end > start + MAX_SECONDS) {
+        end = start + MAX_SECONDS;
```

Comment:
Is there any reason why we have this limitation here? It might be useful to show updates for a full year, for instance, which would be more than MAX_SECONDS.

Reply:
just wanted to reduce the server load (especially for accidental thing), but if there's anything useful, I'm fine to remove the limitation.